src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -329,15 +333,19 @@ private function getSchema(): Schema
         return $schema;
     }
 
-    public static function formatDateTime(\DateTimeInterface $dateTime)
-    {
-        return $dateTime->format('Y-m-d\TH:i:s');
-    }
-
     private function decodeEnvelopeHeaders(array $doctrineEnvelope): array
     {
         $doctrineEnvelope['headers'] = json_decode($doctrineEnvelope['headers'], true);
 
         return $doctrineEnvelope;
     }
+
+    private function getDateTimeType()
+    {
+        if (!class_exists('\Doctrine\DBAL\Types\Types')) {
+            return Type::DATETIME;
```

Comment:
this one seems to be deprecated. so it's good that you used the new constant. but `getSchema` still uses hard-codes the deprecated constant. so it needs to be replaced there as well.

Reply:
If I do not use this method, tests does not passes.
Don't care?

Reply:
please fix the tests then

Reply:
I can't it comes from versions used by travis.
I will remove this and you will see :)

Reply:
You must have misunderstood me. `getDateTimeType` using the fallback was fine. But the method was not used consistently everywhere. I don't see why you removed this now.

Reply:
@Tobion I don't understand what you are asking here. Can you clarify?

Reply:
https://github.com/symfony/symfony/pull/32456#discussion_r329619820